scripts/process_licenses.py

```diff
@@ -61,6 +61,8 @@
     "org.bouncycastle:bcpkix-jdk18on": BOUNCY_CASTLE_LICENSE,
     "org.bouncycastle:bcutil-jdk18on": BOUNCY_CASTLE_LICENSE,
     "org.bouncycastle:bcprov-jdk18on": BOUNCY_CASTLE_LICENSE,
+    "org.roaringbitmap:RoaringBitmap": APACHE_LICENSE,
```

Comment:
can't find this library being used anywhere, why is this change needed?

Reply:
It's a transitive dependency from `org.apache.iceberg:iceberg-core`.

```
[INFO] +- org.apache.iceberg:iceberg-core:jar:1.3.1:compile
[INFO] |  +- org.apache.iceberg:iceberg-common:jar:1.3.1:runtime
[INFO] |  +- org.apache.avro:avro:jar:1.11.1:runtime
[INFO] |  +- org.apache.httpcomponents.client5:httpclient5:jar:5.2.1:runtime
[INFO] |  |  +- org.apache.httpcomponents.core5:httpcore5:jar:5.2:runtime
[INFO] |  |  \- org.apache.httpcomponents.core5:httpcore5-h2:jar:5.2:runtime
[INFO] |  \- org.roaringbitmap:RoaringBitmap:jar:0.9.44:runtime
[INFO] |     \- org.roaringbitmap:shims:jar:0.9.44:runtime
```